### PR TITLE
Resolving error in Register.js

### PR DESCRIPTION
**Pull request by Shivansh-Malhotra**

### CODE REVIEW

File: client/src/components/auth/Register.js [hunk 1]
```diff
@@ -45,18 +45,23 @@ class Register extends Component {
       if ( value.length != 11) {
         msg = 'Enrollment Number must be  11 digits';
       } else {
```
Comment (vtechguys) [inline client/src/components/auth/Register.js]: `Number.isNaN(numEquivalentOfEnrollNo)`
**Required check.**
 I can bypass the input type number. So the user can give a string which is not a valid number. Thus **isNaN** check is always required for the _string to number conversion_ though the **typeof** check is unnecessary here. Use `Number.isNaN` which is more strict than just `isNaN.`

Comment (Shivansh-Malhotra) [reply client/src/components/auth/Register.js]: `else {
        const numEquivalentOfEnrollNo = Number(value);
        if ( Number.isNaN(numEquivalentOfEnrollNo)) {
          msg = "Enrollment Number is not valid number";
        } else {`
You suggest writing like this?

Comment (vtechguys) [reply client/src/components/auth/Register.js]: Yes.
**Number(value) returns valid number (1, 5, 9.8,....) or a invalid number(NaN).**
_Input box always returns the value in form of string_, a string may be "123" or "onetwothree".  Number("123") returns 123 a valid number but Number("onetwothree") returns NaN. Thus before submitting the form, we have to check if the enrollment number is actually a valid number. So the check is required. 
```
const numEquivalentOfEnrollNo = Number(value);
if ( Number.isNaN(numEquivalentOfEnrollNo)) {...} // not valid number
else {...} // valid number further checks
```

**Comment by vtechguys** [comment]
@RobustInfinity Create development servers on your Heroku for frontend and backend

**Pull request merged**